RecoLocalTracker/SubCollectionProducers/src/TrackClusterRemover.cc

```diff
@@ -51,8 +51,8 @@ namespace {
     const TrackCollectionTokens trajectories_;
     edm::EDGetTokenT<QualityMaskCollection> srcQuals;
     
-    const edm::EDGetTokenT<edmNew::DetSetVector<SiPixelCluster> > pixelClusters_;
-    const edm::EDGetTokenT<edmNew::DetSetVector<SiStripCluster> > stripClusters_;
+    edm::EDGetTokenT<edmNew::DetSetVector<SiPixelCluster> > pixelClusters_;
```

Comment:
they can no longer be `const` because they are assigned to in the constructor, instead of being initialised in the initialisation list